tests/EndToEndBuildTests/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <PropertyGroup>
+    <UseFSharpBuild Condition = " '$(UseFSharpBuild)' == '' ">net40</UseFSharpBuild>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('FSharpTests.Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
Since you're pulling in `FSharpTests.Directory.Build.props`, do you even need the breakout of `FSharp.Directory.Build.props` into the `FSharpBuild.*` and `FSharpTests.*` version?  Since the whole reason `FSharpTests.*` exists is to enable these tests, I'd rather the contents of that file move here and simply don't add this line to pull in the parent stuff.